### PR TITLE
Trivial style fixes

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

File: lib/datastruct/kvldskey.h [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 #include "ctassert.h"
 
+/* Key data. */
```
Comment (cperciva) [inline lib/datastruct/kvldskey.h]: `/* Structure for KVLDS key or value (0-255 bytes). */`

File: lib/http/http.h [hunk 2]
```diff
@@ -7,11 +7,13 @@
 /* Opaque type. */
 struct sock_addr;
 
+/* Header data. */
```
Comment (cperciva) [inline lib/http/http.h]: `/* One HTTP header. */`

**Comment by cperciva** [comment]
Looks good, please rebase.

**Comment by gperciva** [comment]
Rebased, ready for merge.

**Pull request merged**